psycopg/psycopg/connection_async.py

```diff
@@ -35,6 +35,22 @@
 logger = logging.getLogger("psycopg")
 
 
+class AsyncPipeline(BasePipeline):
```

Comment:
It doesn't seem there is really a need for Sync/Async pipeline at the moment? I guess it would be different if there were Queues instead of Deque and some blocking was involved here, but it seems that blocking only happens in the connection wait.

Reply:
Done in 5465b9248d331fc9871adc7a61ce88db08417aa1.

Reply:
Cool! It's not impossible that in the future we will split them: it depends if they acquire blocking code